harness/determined/core/_train.py

```diff
@@ -16,6 +19,19 @@ class EarlyExitReason(enum.Enum):
     USER_REQUESTED_STOP = "EXITED_REASON_USER_REQUESTED_STOP"
 
 
+class TensorboardSyncMode(enum.Enum):
+    """
+    ```TensorboardSyncMode`` defines how Tensorboard metrics and profiling data are retained.
+    In ``Auto`` mode only the chief uploads its metrics and profiling data to checkpoint
+    storage. This is the same behavior that existed prior to 0.18.2.
+    In ``Manual`` mode neither metrics nor profiling data are reported. It is up to the user
+    to call ``TrainContext.upload_tensorboard_files()``
+    """
+
+    Auto = "AUTO"
+    Manual = "MANUAL"
```

Comment:
python enums should be in all-caps.

(I see that the ones I recently added are not, but I also see that python enums support [aliasing](https://docs.python.org/3/library/enum.html#duplicating-enum-members-and-values) so I'll make a PR to correct that in a non-breaking way)